LEGO1/mxtimer.cpp

```diff
@@ -10,17 +10,18 @@ long MxTimer::s_LastTimeTimerStarted = 0;
 
 // OFFSET: LEGO1 0x100ae060
 MxTimer::MxTimer()
-{
-  this->m_isRunning = MX_FALSE;
-  MxTimer::s_LastTimeCalculated = timeGetTime();
-  this->m_startTime = MxTimer::s_LastTimeCalculated;
+{
+  this->m_isRunning = MX_FALSE;
+  m_startTime = timeGetTime();
+  // yeah this is somehow what the asm is
+  s_LastTimeCalculated = m_startTime;

```

Comment:
Does the following generate the same asm?
This looks like something that the original devs could have written.
```c++
  s_LastTimeCalculated = m_startTime = timeGetTime();

```

Reply:
While trying to generate a 100% match I noticed some comment (?) made it match